### PR TITLE
PR: fix local development issue

### LINKED ISSUE #569: Local development issue
When I run the bot locally using webhook proxy, it only works for the first webhook received after that it doesn't respond anymore. For example when I start with `yarn start` and post a comment, the bot receives the webhook and processes it, but then I post another comment and the bot doesn't get the webhook. It's really frustrating during development because you have to start and stop the bot for every event.
The webhook proxy might the problem but needs more investigating.

### PR DESCRIPTION
**Pull request by web4er**
Resolves #569
QA: https://github.com/web4er/ubiquibot/issues/54

### CODE REVIEW

**Comment by web4er** [comment]
Another pull request https://github.com/ubiquity/ubiquibot/pull/576 that tried to fix https://github.com/ubiquity/ubiquibot/issues/479 performed similar fixes as it turned to be the same problem for local development and QA bot. I believe only of of these pull requests will be needed to merge.

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "build:ci": "ncc build src/adapters/github/github-actions.ts -o ./",
     "build:serverless": "ncc build src/index.ts -o ./",
-    "build": "tsc",
+    "build": "tsc && copyfiles src/assets/images/* lib/",
```
Comment (pavlovcik) [inline package.json:16]: Maybe cleaner to make a new `postbuild` script. 

https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts

**Review by pavlovcik — CHANGES_REQUESTED** [review]

**Review by pavlovcik — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "build:ci": "ncc build src/adapters/github/github-actions.ts -o ./",
     "build:serverless": "ncc build src/index.ts -o ./",
     "build": "tsc",
+    "postbuild": "copyfiles src/assets/images/* lib/",
```
Comment (pavlovcik) [inline package.json:17]: I hope you verified that this works because I've had issues on occasion with the pre/post scripts.

Comment (web4er) [reply package.json:17]: Yes. I ran it on the local, and then also performed some tests.

**Pull request merged**